bin/mail/2016-11-03-hoc-global-site-refresh/generate-teachers

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+
+# 3 Categories (in order), represented by a category param:
+# 1. signup2016:
+#   - HocSignup2016
+# 2. past_organizer
+#   - HocHardwarePrizes2014
+#   - HocHardwarePrizes2015
+#   - HocSignup2014
+#   - HocSignup2015
+#   - HocSignup2016
+#   - HocSurvey2014
+#   - HocSurvey2015
+#   - CSEdWeekEvent2013
+# 3. teacher
+#   - Code Studio teachers
+#   - Petition (educators)
+#   - BringToSchool2013
+#   - ClassSubmission
+#   - DistrictPartnerSubmission
+#   - PLP interest form
+#   - Teacher interest form
+#   - School interest form
+#   - HelpUs2013
+#   - K5OnlineProfessionalDevelopmentPostSurvey
+#   - K5ProfessionalDevelopmentSurvey
+#   - ProfessionalDevelopmentWorkshop
+#   - ProfessionalDevelopmentWorkshopSignup
+#   - StudentNomination
```

Comment:
TIL SOTM is a thing :) It looks like they're nominated by teachers, mentors, etc according to [this](https://github.com/code-dot-org/code-dot-org/blob/a9751f0826f84db68fdd761902e60ccbb3d02e48/pegasus/sites.v3/code.org/views/nominate_student_form.haml#L11-L11). Incidentally it looks like we haven't had a new SOTM since [June](https://code.org/sotm) (click See all).